### DISCUSSION TITLE
Slack notifications should include more information about the latest change

### DISCUSSION THREAD

**Discussion by rachellawson** [GitbookIO/community#103] [Feature Requests]
### What's your feature/idea?

I’m liking the idea of Slack notifications when content changes in the associated space, however, it is difficult to understand what the change relates to, just from the message. 

A recently read message says:

> Content of **Development** has been updated

Which is great but Development is the name of the space inside the ‘product-committee’ collection and I don’t know which change this is without going to look.

### What's your desired outcome?

I would change the text to include more information, including the collection it belongs to, the space, and the name of the change request. Something like:

> Content of **product-committee**/**Development** has changed. The change request was **’PRD-105 Update images to use Mermaid.js’**

(Where product-committee is the collection name and development is the space name)

All the information above should be available to the bot so shouldn’t be too big a job, tbh. Hey - you could even throw in the approver’s userID if you like!

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

In the future, it might be even better to allow a bit of control over the text - like a small template

**Comment by MantisClone** [comment]
Agreed! The Gitbook Slack notification should include links to the:

1. Collection
2. Space
3. Change Request

Author and Approver's userID would be cool too!

The current notification leaves a lot to be desired.
![image](https://github.com/GitbookIO/community/assets/2530913/96ae1851-b5f7-462c-8f13-35d0ec4d3644)

**Reply by KristianZ** [reply]
^^^^ 
Agreed! We found the Slack extension not very informative for large documentation websites. Including at least collection, space, author, and change request information would be ideal

**Comment by denisemcg** [comment]
I've just set up this integration and it works well. It tells users what page was modified, added or deleted. In the case of a modified page, it would be great to see exactly what was changed, maybe a before and after view, or link to a specific section on the page.
